flink-table/flink-table-planner/src/test/java/org/apache/flink/table/api/CompiledPlanITCase.java

```diff
@@ -404,28 +404,13 @@ void testPersistedConfigOption() throws Exception {
     }
 
     @Test
-    void testBatchMode() {
+    public void testExecutePlanBatchMode() throws Exception {
```

Comment:
`public` is not required

Reply:
rename method to `testCompileAndExecutePlanBatchMode` to match with the surrounding methods

Reply:
Done!